src/sidebar/components/test/MentionSuggestionsPopover-test.js

```diff
@@ -90,4 +91,39 @@ describe('MentionSuggestionsPopover', () => {
       assert.calledWith(onSelectUser, defaultUsers[index]);
     });
   });
+
+  [
+    {
+      mentionMode: 'username',
+      shouldShowUsernames: true,
+    },
+    {
+      mentionMode: 'display-name',
+      shouldShowUsernames: false,
+    },
+  ].forEach(({ mentionMode, shouldShowUsernames }) => {
+    it('renders only display names only when mention mode is username', () => {
```

Comment:
This sentence doesn't make sense. Perhaps "renders expected suggestions according to mention mode"?

Reply:
> This sentence doesn't make sense. 

OMG. Indeed it doesn't 😂